glue-lwip/esp-dhcpserver.c

```diff
@@ -109,6 +109,54 @@ void ICACHE_FLASH_ATTR node_remove_from_list(list_node **phead, list_node* pdele
         }
     }
 }
+
+/******************************************************************************
+ * FunctionName : wifi_softap_add_dhcps_lease
+ * Description  : add static lease on the list, this will be the next available @
+ * Parameters   : mac address
+ * Returns      : true if ok and false if this mac already exist or if all ip are already reserved
+*******************************************************************************/
+bool ICACHE_FLASH_ATTR wifi_softap_add_dhcps_lease(uint8 *macaddr)
+{
+
+	struct dhcps_pool *pdhcps_pool = NULL;
+	list_node *pback_node = NULL;
+
+	uint32 start_ip = dhcps_lease.start_ip.addr;
+	uint32 end_ip = dhcps_lease.end_ip.addr;
+
+	for (pback_node = plist; pback_node != NULL;pback_node = pback_node->pnext) {
+	pdhcps_pool = pback_node->pnode;

```

Comment:
(I know this file is not a model of clean C)
Can you please indent this inner for block ?

Reply:
Just updated, 
I tryed to reproduce indent as the rest of the file